RetailCoder.VBE/Inspections/Resources/InspectionsUI.resx

```diff
@@ -647,7 +647,7 @@ If the parameter can be null, ignore this inspection result; passing a null valu
   <data name="ApplicationWorksheetFunctionQuickFix" xml:space="preserve">
     <value>Use Application.WorksheetFunction explicitly.</value>
   </data>
-  <data name="AssignedByValParameterQuickFix" xml:space="preserve">
+  <data name="AssignedByValParameterMakeLocalCopyQuickFix" xml:space="preserve">
```

Comment:
If they don't exist yet, you may as well leave them missing ... Because then the translators for the respective languages (me and retailcoder) need to get to those either way. On that note: @retailcoder we should probably do a i18n check for the next release either way ...